signal_op/extract_candidate_events.m

```diff
@@ -1,11 +1,10 @@
-function idx_event_frames = extract_candidate_events(trace,varargin)
+function idx_event_frames = extract_candidate_events(trace,mad_scale,varargin)
 % Extract maximum points in the trace as candidate events
 % varargin : 1 optional argument. Set it to 1 if you want to plot the
 % output
```

Comment:
The display option (when the function is invoked with `varargin==1`) should provide more detail to what we're seeing, i.e. a legend (like you provide in `extract_events`).